website/docs/troubleshooting/login.md

```diff
@@ -27,3 +27,29 @@ uv run ak create_recovery_key 10 akadmin
 ```
 
 This will output a link, that can be used to instantly gain access to authentik as the user specified above. The link is valid for amount of years specified above, in this case, 10 years.
+
+## Can't access initial setup
+
+If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers as this often resolves temporary issues.
```

Comment:
```suggestion
If you're unable to access the initial setup flow (`/if/flow/initial-setup/`), first try restarting the containers because this often resolves temporary issues.
```